cmd/main.go

```diff
@@ -125,42 +125,32 @@ func main() {
 		log.Fatal(err.Error())
 	}
 
+	mustSucceedf := func(err error, msg string, args ...interface{}) {
```

Comment:
perhaps choose a clearer name such a callOrAbort or abortOnFailure

Reply:
it's always call anything, maybe failOnError?